src/utils/abbreviations.js

```diff
@@ -0,0 +1,314 @@
+// Abbreviations starting with '|' don't receive a delimeter (eg. '.') after them
+const nonAbbreviator = '|';
```

Comment:
I think the nonAbbreviator makes this a bit unclear; perhaps export two separate lists (one that should have the dot at the end, and one that shouldn't?).